### ISSUE TITLE
Inconsistent Crash when Liberty Navy Heavy Fighter NPC is shot down

### ISSUE THREAD

**Issue by IrateRedKite** [better-modernized-combat/bmod-client#91]
This was occurring in Silverton, and was likely lootprops related. Unfortunately was unable to recover server logs for the event, but this will need finding and fixing.

**Comment by IrateRedKite** [comment]
Bumping the priority of this one up as it hampers live testing. I am not sure this is just the Liberty HF. The server is crashing in KernelBase.dll with the following:

```
Faulting application name: FLServer.exe, version: 0.1.1254.11, time stamp: 0x3ecbb13e
Faulting module name: KERNELBASE.dll, version: 10.0.17763.5696, time stamp: 0x31f3be07
Exception code: 0xe06d7363
Fault offset: 0x00126122
Faulting process id: 0x150c
Faulting application start time: 0x01daa359989b811b
Faulting application path: C:\bmod-server\Freelancer\EXE\FLServer.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: 13cd65c7-1c0f-42b3-a5ec-1ef2c8369059
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by IrateRedKite** [comment]
```
[2024-05-11 20:16:04.555] [FLHook] [error] Exception in IServerImplHook::SPMunitionCollision on server call
[2024-05-11 20:16:04.556] [FLHook] [error] ERROR: SEH Exception in IServerImplHook::SPMunitionCollision on line 4390; minidump may contain more information.
[2024-05-11 20:16:43.405] [FLHook] [critical] !!TOP LEVEL EXCEPTION!!
```

```
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xb7c1ae49) missing hardpoint ({bm_placeholder_ship}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0x91dfa98c) missing hardpoint ({bm_li_hf_li_n_grp_elite}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0x91dfa98c) missing hardpoint ({bm_li_hf_li_n_grp_pilot}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0x91dfa98c) missing hardpoint ({bm_li_hf_li_n_grp_rookie}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0x91dfa98c) missing hardpoint ({bm_li_hf_li_n_grp_pilot}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xb7c1ae49) missing hardpoint ({bm_placeholder_ship}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xa987e580) missing hardpoint ({bm_cv_hf_fc_x_grp_rookie}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xa987e580) missing hardpoint ({bm_cv_hf_fc_x_grp_rookie}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xb7c1ae49) missing hardpoint ({bm_placeholder_ship}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xb7c1ae49) missing hardpoint ({bm_placeholder_ship}) for item (0x81837304)
E:\FL\Scratch\Source\Server\Space\EqObj.cpp(2519) : *** ERROR: ID(0x6C4) Arch(0xa987e580): Tried to apply damage of type ??? within another damage callback.
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xb7c1ae49) missing hardpoint ({bm_placeholder_ship}) for item (0x81837304)
E:\FL\Scratch\Source\Common\CExternalEquip.cpp(142) : *** WARNING: CExternalEquip: Object (0xb7c1ae49) missing hardpoint ({bm_placeholder_ship}) for item (0x81837304)
```

Occurred when destroying(?) a rookie in Cheyenne. I am fairly confident this is a lootprops/ammo issue

**Comment by IrateRedKite** [comment]
This may have been fixed with https://github.com/better-modernized-combat/bmod-client/pull/105, moving to in review

**Comment by IrateRedKite** [comment]
Closed with f23782d7b9db175e3845bf2a80b2c72ec5ef74b3
